dtmi/fii/fii/badge_apt-1.json

```diff
@@ -0,0 +1,30 @@
+{
+    "@id": "dtmi:fii:fii:badge_apt;1",
+    "@type": "Interface",
+    "contents": [
+        {
+            "@type": "Telemetry",
+            "displayName": {
+                "en": "Battery Level"
+            },
+            "description": "unit is %",

```

Comment:
As displayName and description are used in UX we recommend putting clear descriptions with good grammar (i.e. capitalizing first letter in sentence).

Not a strict requirement though.

Reply:
I have update model file to follow UX rule.